src/time/Storage.hpp

```diff
@@ -23,7 +24,7 @@ class Storage {
    *
    * This Storage is used in the context of Waveform relaxation where samples in time are provided. Starting at the beginning of the window with time 0.0 and reaching the end of the window with time 1.0.
    */
-  Storage();
+  Storage(int _interpolationOrder);
```

Comment:
Do we really need to change the constructor here to accept `_interpolationOrder`? (btw: Please write `interpolationOrder` for this argument name. The `_variableName` is reserved for class members.

If possible, please use `Storage::setInterpolationOrder`. I remember that I introduced this function, because at the time where we have to construct the `Storage` (somewhere in the configuration), we do not know the interpolation degree yet (somewhere *later* in the configuration).

Also important: Please write `interpolationDegree`, not `interpolationOrder`. We fixed the naming in https://github.com/precice/precice/issues/1699, because we actually talk about the B-Spline degree and not necessarily about the order or interpolation scheme.